include/dmtcp.h

```diff
@@ -262,6 +262,34 @@ int dmtcp_enable_ckpt(void) __attribute__((weak));
 #define dmtcp_enable_ckpt() \
   (dmtcp_enable_ckpt ? dmtcp_enable_ckpt() : DMTCP_NOT_PRESENT)
 
+/**
+ * Example:  dmtcp_get_libc_addr("fork") returns the address of "fork"
+ *           in libc in memory.
```

Comment:
in libc in memory? Were you intending to write that it returns the address of "fork" in libc loaded in memory?